homeassistant/components/zha/.translations/pl.json

```diff
@@ -12,7 +12,6 @@
                     "radio_type": "Typ radia",
                     "usb_path": "\u015acie\u017cka urz\u0105dzenia USB"
                 },
-                "description": "Puste",
```

Comment:
This file is generated by the download script. If we change it in this PR, the next time we update the translations, it will be overridden again.

Reply:
I think this indicates that the key should be removed from Lokalise.co

Reply:
Please remove this from this PR and I will do that in Lokalise.co

Reply:
Okay, or maybe I should just rebase after your PR is merged

Reply:
Rebased and fixed